VERSION.south.playback

```diff
@@ -1,2 +1,2 @@
-foglamp_south_playback_version=1.6.0
+foglamp_south_playback_version=1.7.0
 foglamp_version>=1.6
```

Comment:
I would suggest we make the >=1.7 as we have not tested the plugin against the 1.6 release

Reply:
There is no such change But anyway I will do that to be sure.